pkgs/by-name/un/unrar-free/setup-hook.sh

```diff
@@ -0,0 +1,5 @@
+unpackCmdHooks+=(_tryUnrarFree)
+_tryUnrarFree() {
+    if ! [[ "$curSrc" =~ \.rar$ ]]; then return 1; fi
+    unrar-free x "$curSrc" >/dev/null
```

Comment:
Will not work:

```
$ ./result/bin/unrar-free --help
Usage: unrar-free [OPTION...] ARCHIVE [FILE...] [DESTINATION]
Extract files from rar archives.

  -P, --print                Print archive files to stdout
  -x, --extract              Extract files from archive (default)
  -t, --list                 List files in archive
  -f, --force                Overwrite files when extracting
      --extract-newer        Only extract newer files from the archive
      --extract-no-paths     Don't create directories while extracting
  -i, --outfile=FILE         File to print the messages to (nul is short for
                             /dev/null, err is short for /dev/stderr)
  -p, --password             Decrypt archive using a password
  -?, --help                 Give this help list
      --usage                Give a short usage message
  -V, --version              Print program version
```

And yes, sadly, `unrar-free` does not support flags without a `-``.

Reply:
I don't know how I missed that... I thought I'd tried this and it worked, but it's fixed now either way.